go.mod

```diff
@@ -8,18 +8,22 @@ require (
 	github.com/cenkalti/backoff/v4 v4.1.1
 	github.com/fatih/color v1.12.0
 	github.com/ghodss/yaml v1.0.0
+	github.com/go-openapi/spec v0.19.5 // indirect
 	github.com/google/go-querystring v1.1.0
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-memdb v1.3.2
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/imdario/mergo v0.3.12
-	github.com/kong/go-kong v0.20.0
+	github.com/kong/go-kong v0.21.0
+	github.com/kr/pty v1.1.5 // indirect
 	github.com/mitchellh/go-homedir v1.1.0
+	github.com/satori/go.uuid v1.2.0 // indirect
```

Comment:
Why is this here? Did you forget to tidy the module?

Reply:
Apparently so. Weird that we don't have a lint failure on that here. 938d8dc